src/common/Utilities.cpp

```diff
@@ -161,18 +161,18 @@ const char *convertToText<StreamState>(StreamState state) {
 }
 
 template<>
-const char *convertToText<AudioApi >(AudioApi audioApi) {
+const char *convertToText<AudioApi>(AudioApi audioApi) {
 
     switch (audioApi) {
         case AudioApi::Unspecified: return "Unspecified";
         case AudioApi::OpenSLES:    return "OpenSLES";
         case AudioApi::AAudio:      return "AAudio";
-        default:                    return "Unrecognised audio API";
+        default:                    return "Unrecognized audio API";
```

Comment:
Interesting. The word "unrecogniSed" appears 838 times in the Android code base while "unregogniZed" appears 9435 times. Let's go with "Zed".